test/e2e/e2e_test.go

```diff
@@ -361,28 +364,37 @@ func TestLowNodeUtilization(t *testing.T) {
 	}
 	podsBefore := len(podsOnMosttUtilizedNode)
 
-	t.Log("Running LowNodeUtilization strategy")
-	nodeutilization.LowNodeUtilization(
-		ctx,
-		clientSet,
-		deschedulerapi.DeschedulerStrategy{
-			Enabled: true,
-			Params: &deschedulerapi.StrategyParameters{
-				NodeResourceUtilizationThresholds: &deschedulerapi.NodeResourceUtilizationThresholds{
-					Thresholds: deschedulerapi.ResourceThresholds{
-						v1.ResourceCPU: 70,
-					},
-					TargetThresholds: deschedulerapi.ResourceThresholds{
-						v1.ResourceCPU: 80,
-					},
-				},
-			},
+	t.Log("Running LowNodeUtilization plugin")
+
+	handle := &frameworkfake.HandleImpl{
+		ClientsetImpl:                 clientSet,
+		GetPodsAssignedToNodeFuncImpl: getPodsAssignedToNode,
+		PodEvictorImpl:                podEvictor,
+		EvictorFilterImpl: evictions.NewEvictorFilter(
+			workerNodes,
+			getPodsAssignedToNode,
+			true,
+			false,
+			false,
+			false,
+			evictions.WithNodeFit(false),
```

Comment:
This line can be omitted as `false` is the default value for the NodeFit.

Reply:
update now.